recipe/meta.yaml

```diff
@@ -23,36 +27,61 @@ build:
 outputs: 
   - name: {{ name }}-examples
     build:
-      script: "%RECIPE_DIR%/build-examples.bat"
+      script: "%RECIPE_DIR%/build-examples.bat" #[win]
+      script: "$RECIPE_DIR/build-examples.sh" #[not win]
 
   - name: {{ name }}-documentation
     build:
-      script: "%RECIPE_DIR%/build-documentation.bat"
+      script: "%RECIPE_DIR%/build-documentation.bat" #[win]
+      script: "$RECIPE_DIR/build-documentation.sh" #[not win]
   
   - name: {{ name }}
     build:
-      script: "%RECIPE_DIR%/build-telemac.bat"
+      script: "%RECIPE_DIR%/build-telemac.bat" #[win]
     requirements:
-      build:
-        - m2-base
-        - m2w64-metis
-        - m2w64-msmpi
-        # - t2d-msmpi
-        # - m2w64-mumps =5.2.1
-        # - m2w64-scalapack
-        - m2w64-toolchain
+      host:
+        - m2-base #[win]
+        - m2w64-metis #[win]
+        - m2w64-msmpi #[win]
+        # - t2d-msmpi #[win]
+        # - m2w64-mumps =5.2.1 #[win]
+        # - m2w64-scalapack #[win]
+        - m2w64-toolchain #[win]
+        - {{ compiler('fortran') }} #[not win]
+        - {{ mpi }} #[not win]
+        - {{ mpi }}-mpifort #[not win]
+        - gcc #[not win]
+        - binutils #[not win]
+        - hdf5 #[not win]
+        - metis #[not win]
+        - libmed #[not win]
+        - mumps-mpi #[not win]
+        - scalapack #[not win]
+        - parmetis #[not win]
         - matplotlib
         - numpy <1.22.0
         - python <3.11
         - scipy <1.11
       run:
-        - m2w64-metis
-        - m2w64-msmpi
-        # - t2d-msmpi
-        # - m2w64-mumps =5.2.1
-        # - m2w64-scalapack
-        - m2w64-toolchain
+        - m2w64-metis #[win]
+        - m2w64-msmpi #[win]
+        # - t2d-msmpi #[win]
+        # - m2w64-mumps =5.2.1 #[win]
+        # - m2w64-scalapack #[win]
+        - m2w64-toolchain #[win]
+        - {{ compiler('fortran') }} #[not win]
+        - {{ mpi }} #[not win]
+        - {{ mpi }}-mpifort #[not win]
+        - gcc #[not win]
+        - binutils #[not win]
+        - hdf5 #[not win]
+        - metis #[not win]
+        - libmed #[not win]
+        - mumps-mpi #[not win]
+        - scalapack #[not win]
+        - parmetis #[not win]
         - matplotlib
+        - mpi4py
```

Comment:
We could add jupyter and all other dependencies to correctly run notebooks

Reply:
ok done
added
```
        - jupyter-lab
        - ipympl
        - ipykernel
```